packages/blockchain-link-types/src/responses.ts

```diff
@@ -92,12 +93,22 @@ export interface GetFiatRatesTickersList {
     };
 }
 
+export interface PriorityFeeEstimationDetails {
+    maxFeePerGas: string;
+    maxPriorityFeePerGas: string;
+    minWaitTimeEstimate?: number;
+    maxWaitTimeEstimate?: number;
+}
+
+export type FeeTrend = 'up' | 'down';
```

Comment:
```suggestion
```

probably unused, if used, you can use `Eip1559Fees['baseFeeTrend']` or `Eip1559Fees['priorityFeeTrend']`